File: Day 44/recursiveSum.js
function recursiveAdd(){

    console.log('We will sum the number you enter')

    let number = parseInt( prompt('Type a number to sum. Type 0 to finish: ') )

    if(isNaN(number)){
        console.log('This is not a number!')
        return recursiveAdd()
    }
    else{
        
        if (number !== 0) 
            return number + recursiveAdd()
        
        else
            return number
        

    }
    
}

recursiveAdd()

File: Day 44/recursiveSum.c
#include <stdio.h>
#include <math.h>

int recursiveAdd();

int main()
{
    int total = recursiveAdd();
    printf("The total sum is %d \n", total);

    return 0;
}

int recursiveAdd(){


    int number;
    printf("Type a number to sum: \n");
    scanf("%d", &number);

    if( number != 0 )
        return number + recursiveAdd();
    else
        return number;
    
}